tests/test_scylla.c

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright (C) 2019 BiiLabs Co., Ltd. and Contributors
+ * All Rights Reserved.
+ * This is free software; you can redistribute it and/or modify it under the
+ * terms of the MIT license. A copy of the license can be found in the file
+ * "LICENSE" at the root of this distribution.
+ */
+
+#include "storage/scylla_api.h"
+#include "test_define.h"
+
+static char* host;
+
+void test_get_column_from_edgeTable(CassSession* session) {
+  char expected_result[][FLEX_TRIT_SIZE_243] = {
+      {BUNDLE_1},       // address_2
+      {BUNDLE_2},       // address_2
+      {BUNDLE_3},       // address_2
+      {BUNDLE_1},       // address_1
+      {TRANSACTION_4},  // approvees T7
+      {TRANSACTION_3},  // approvees T7
+      {TRANSACTION_6},  // approvees T7
+      {TRANSACTION_5}   // approvees T7
+  };
+  hash243_queue_t res_hash = NULL;
+  hash243_queue_entry_t* iter243 = NULL;
+  get_column_from_edgeTable(session, &res_hash, (cass_byte_t*)ADDRESS_2, "bundle");
+  get_column_from_edgeTable(session, &res_hash, (cass_byte_t*)ADDRESS_1, "bundle");
+  get_column_from_edgeTable(session, &res_hash, (cass_byte_t*)TRANSACTION_7, "hash");
+  size_t idx = 0;
+  CDL_FOREACH(res_hash, iter243) {
+    TEST_ASSERT_FALSE(
+        memcmp(iter243->hash, (flex_trit_t*)expected_result[idx++], sizeof(flex_trit_t) * FLEX_TRIT_SIZE_243));
+  }
+  hash243_queue_free(&res_hash);
+  TEST_ASSERT_EQUAL_INT(idx, sizeof(expected_result) / (sizeof(char) * FLEX_TRIT_SIZE_243));
+
+  return;
```

Comment:
This one is redundant

Reply:
Without this, if we get fewer results then expected, the test will still pass.

Reply:
why0.0?

Reply:
Maybe try to include "unity.h"?

Reply:
Actually what I need is hash_queue_t, not sure where it is.
I need tp search it for a while.

Reply:
I dont get it sor

Reply:
I think this discussion mix two things.


> Without this, if we get fewer results then expected, the test will still pass.

I mention it for `TEST_ASSERT_EQUAL_INT(idx, sizeof(expected_result) / (sizeof(char) * FLEX_TRIT_SIZE_243));`

> Actually what I need is hash_queue_t, not sure where it is.
> I need tp search it for a while.

This is talking about the header, I  include accelerator/config.h for hash_queue_t and status_t

Reply:
I mean maybe we could solve this problem by include `unity.h` and remove the `return`

Reply:
OK! I've revised to more specific header.